src/convolution.cpp

```diff
@@ -510,9 +510,7 @@ std::size_t ConvolutionDescriptor::BackwardWeightsGetWorkSpaceSizeGEMM(
         return *std::max_element(gemm_ws_szs.begin(), gemm_ws_szs.end());
     }
 #else
-    std::ignore = dyDesc;
```

Comment:
[Post Merge Review]
This issue was introduced in #852 and fixed with #1058
JIRA reported at https://ontrack-internal.amd.com/browse/SWDEV-301586 
Mainline: 
from 791b419c49485353c0e70ccb92be26bcdcf97e9c (05/18/2021)
fixed in 1e90891903ad795a3928cde443ed1921f9c9de52 (latest of as 09/02/2021)